### PR TITLE
Features

### PR DESCRIPTION
**Pull request by GrantHuang0101**

### CODE REVIEW

File: src/views/dashborad/transactions/CreateTransaction.jsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  const [previews, setPreviews] = useState([]);
+
+  const [selectedDate, setSelectedDate] = useState(
+    format(new Date(), "yyyy-MM-dd")
+  );
+  const [description, setDescription] = useState("");
+
+  useEffect(() => {
+    axios
+      .get("http://localhost:8080/accounts", {
```
Comment (eric-cw-hsu) [inline src/views/dashborad/transactions/CreateTransaction.jsx]: remove domain

File: src/views/dashborad/transactions/CreateTransaction.jsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+          },
+        })
+        .then((response) => {
+          console.log("Post response:", response.data);
+        })
+        .catch((error) => {
+          console.log(error);
+        });
+    }
+    navigate("/user/transactions");
```
Comment (eric-cw-hsu) [inline src/views/dashborad/transactions/CreateTransaction.jsx]: if the fetch fails, it should not navigate

File: src/views/dashborad/transactions/EditTransaction.jsx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    setPreviews((prevPreviews) =>
+      prevPreviews.filter((preview) => preview.rowId !== rowId)
+    );
+  };
+
+  const handlePreview = (rowId, values) => {
+    setPreviews((prevRows) => {
+      const existingIndex = prevRows.findIndex((row) => row.rowId === rowId);
+
+      if (existingIndex !== -1) {
```
Comment (eric-cw-hsu) [inline src/views/dashborad/transactions/EditTransaction.jsx]: NO CONSTANT NUMBER

exec.
const INDEX_NOT_FOUND = -1

if (existingIndex != INDEX_NOT_FOUND)

**Review by eric-cw-hsu — COMMENTED** [review]
Please redo the commit with proper commit message.

File: src/views/dashborad/transactions/EditTransaction.jsx (hunk 3 above)
Comment (GrantHuang0101) [reply src/views/dashborad/transactions/EditTransaction.jsx]: done

**Pull request merged**